dashboard/containers/dispatcher/fontprojects.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import csv
+import urllib
+CACHE_CSV = "fontprojects.csv"
```

Comment:
Why not store this in memory using StringIO?

https://docs.python.org/2/library/stringio.html